portafly/src/i18n/locales/en/product.yml

```diff
@@ -0,0 +1,30 @@
+---
+# ------------------ Product > Create product
+# Page details
+
+page_title: Create product #--- PROPOSED
+page_title_description: Page title to create a product
+body_title: Create a new product #--- PROPOSED
+body_title_description: Body title of the page to create a new product
+
+# Main area
+
+name: Name
+name_description: Field label to show/input the name of the product.
+system_name: System name
```

Comment:
@porueesq according to [UX specs](https://github.com/3scale/porta/blob/master/portafly/doc/ux_specs/product/create_product.md) this field should feature placeholder text and helper text

Reply:
@cntlsn What kind of dashes are allowed in the the system name: en dashes or em dashes?
![image](https://user-images.githubusercontent.com/43811485/88658227-633f6f80-d0d3-11ea-805e-61e962d3dc24.png)

Reply:
@porueesq I have no idea, maybe @josemigallas knows.

Reply:
@porueesq it's actually referring to hyphens I think

Reply:
**hyphen** yes 👍